src/os/windows/win32.cpp

```diff
@@ -419,10 +412,8 @@ int APIENTRY WinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, LPSTR lpCmdLi
 
 	CrashLog::InitialiseCrashLog();
 
-#if defined(UNICODE)
 	/* Check if a win9x user started the win32 version */
-	if (HasBit(GetVersion(), 31)) usererror("This version of OpenTTD doesn't run on windows 95/98/ME.\nPlease download the win9x binary and try again.");
-#endif
+	if (HasBit(GetVersion(), 31)) usererror("OpenTTD 1.9 and later doesn't run on windows 95/98/ME any longer.");
```

Comment:
I don't think a Windows 9x user would even get this far, as our binaries would now have a minimum version of 5.1 anyway. Probably safe to remove this entirely.